api-test/src/main/resources/pms/DeactivateMISPLicense/DeactivateMISPLicense.yml

```
@@ -0,0 +1,20 @@
+DeactivateMISPLicense:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)